ddtrace/internal/telemetry/writer.py

```diff
@@ -160,6 +160,7 @@ def on_shutdown(self):
     def _stop_service(self, *args, **kwargs):
         # type: (...) -> None
         super(TelemetryWriter, self)._stop_service(*args, **kwargs)
+        # TODO: Call this with an atexit hook
```

Comment:
Tagging @mabdinur in case you'd be interested in this

Reply:
Is this an actionable comment that needs to be worked on in this PR, or just an overall `todo` comment with no urgent timeline?